src/TextBox.js

```diff
@@ -120,12 +123,14 @@ export default class TextBox extends BaseClass {
           @private
       */
       function checkSize() {
+        const truncate = () => {
+          if (line < 1) lineData = [truncateWord(wrapResults.words[0], that._ellipsis("", line), w, style)];
+          else lineData[line - 1] = that._ellipsis(lineData[line - 1], line);
+        };
 
-        if (fS < fMin) {
-          lineData = [];
-          return;
-        }
-        else if (fS > fMax) fS = fMax;
+        // Constraint the font size
+        fS = Math.max(fS, fMin);
+        fS = Math.min(fS, fMax);
```

Comment:
I'd prefer to use the `max` and `min` functions imported from `d3-array` instead of the vanilla JS Math functions.

Reply:
Interested to know the reason for this? Happy to make the change but I'm always pro vanilla if possible